.goreleaser.yml

```diff
@@ -0,0 +1,41 @@
+# This is an example goreleaser.yaml file with some sane defaults.
+# Make sure to check the documentation at http://goreleaser.com
+# before:
+#   hooks:
+#     # You may remove this if you don't use go modules.
+#     - go mod download
+#     # you may remove this if you don't need go generate
+#     - go generate ./...
+builds:
+- id: registry
```

Comment:
Let's drop the registry binary from this -- it's primarily for testing and I don't want to implicitly endorse its use by including it in releases.

Reply:
Done